api/ws/workers/middlewares/handshake.js

```diff
@@ -72,11 +72,17 @@ var middleware = {
 		return function(headers, cb) {
 			z_schema.validate(headers, definitions.WSPeerHeaders, error => {
 				if (error) {
+					var errorDescription;
+					if (error[0].path && error[0].path.length) {
```

Comment:
Can be rewritten to 
```
const { path, message } = error[0];
const errorDescription = path ? `${path}: ${message}` : `${message}`;
```

Reply:
That file doesn't use es6 yet, I thought that the priority is to keep consistency at the file level.